_info/Tribute

```diff
@@ -1,6 +1,7 @@
 Tribute
 
-Every morning, the king has to choose to tribute one of this team members. This is mandatory. The king has to also choose a certain amount of money to assign to the person. Each team will do the same. The team which pays the least money for their tribute, will have their tribute killed. Use `!tribute` to set your tribute candidate if you're the king. 
+Every morning, the king has to choose to tribute one of this team members. This is mandatory. The king has to also choose a certain amount of money to assign to the person. Each team will do the same. 
```

Comment:
should probably say day instead of morning, because it's good if we use the game mechanic terminology to explain things whenever possible.

Reply:
Every day makes people think it can be either day or night, morning makes it clear it's during the day phase.

I'll change it to day anyway since you requested it, but that was my logic earlier I think

Reply:
thats true, maybe could say each day phase instead

Reply:
alright will do